builtin/common/settings/dlg_change_mapgen_flags.lua

```diff
@@ -46,33 +46,45 @@ local function get_formspec(dialogdata)
 	end
 
 	local fields = {}
-	local function add_field(x, name, label, value)
+	local function add_field(x, name, label, value, tt)
 		fields[#fields + 1] = ("field[%f,%f;3.3,1;%s;%s;%s]"):format(
 			x, height, name, label, core.formspec_escape(value or "")
 		)
+		if tt then
+			fields[#fields + 1] = ("tooltip[%s;%s]"):format(name, tt)
+		end
 	end
 	-- First row
 	height = height + 0.3
-	add_field(0.3, "te_offset", fgettext("Offset"), t[1])
-	add_field(3.6, "te_scale",  fgettext("Scale"),  t[2])
-	add_field(6.9, "te_seed",   fgettext("Seed"),   t[6])
+	add_field(0.3, "te_offset", fgettext("Offset"), t[1], fgettext("This value offsets the final noise."))
+	add_field(3.6, "te_scale",  fgettext("Scale"),  t[2], fgettext("This value multiplies the noise before the offset is added."))
+	add_field(6.9, "te_seed",   fgettext("Seed"),   t[6], fgettext("This value acts as a random seed for the noise. Every value will result in a different, random result."))
```

Comment:
```suggestion
	add_field(6.9, "te_seed",   fgettext("Seed"),   t[6], fgettext("This value acts as a random seed for the noise. The same seed results in the same base noise."))
```

something like this. that's the important property of seeds: reproducibility. e.g. if i have a noise resulting in cool terrain features in some spot, i might want to pin the seed so someone else can reproduce it.